src/containers/App/App.js

```diff
@@ -1,10 +1,23 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import { NoteArea, CreateNote } from "../";
 
 class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      showPopup: false
+    };
+  }
+  handleClick = () => this.setState({ showPopup: !this.state.showPopup });
+  addNote() {}
   render() {
+    const { showPopup } = this.state;
     return (
       <div className="App">
-        <h1 className='test'>Trapper-Keeper</h1>
+        <h1 className="title">Trapper-Keeper</h1>
+        <div onClick={this.handleClick}>Add Note</div>
+        <CreateNote canRender={showPopup} addNote={this.addNote} />
```

Comment:
If the above function IS leftover, we can probably remove the addNote prop.  Let me know if this is not the case! :)